File: codeforces/low_difficulty/difficulty_900/3.hq9+_[133A]/hq9+.c
#include <stdio.h>
#include <string.h>
#include <stdbool.h>
/* Author: José Rodolfo (jric2002) */
int main() {
  char kw[4] = {'H', 'Q', '9', '+'};
  char p[100];
  unsigned short int size_p, size_kw;
  bool output;
  scanf("%s", &p);
  size_p = strlen(p);
  size_kw = sizeof(kw)/sizeof(char);
  output = false;
  for (unsigned short int i = 0; i < size_p && !output; i++) {
    for (unsigned short int j = 0; j < (size_kw - 1); j++) {
      if (p[i] == kw[j]) {
        output = true;
        break;
      }
    }
  }
  (output) ? printf("YES\n") : printf("NO\n");
  return 0;
}

File: codeforces/low_difficulty/difficulty_900/3.hq9+_[133A]/hq9+.py
# Author: José Rodolfo (jric2002)
kw = ['H', 'Q', '9', '+']
p = str(input())
size_p = len(p)
size_kw = len(kw)
output = False
i = 0
while (i < size_p and not(output)):
  for j in range(0, (size_kw - 1)):
    if (p[i] == kw[j]):
      output = True
      break
  i += 1
print("YES") if (output) else print("NO")